### ISSUE TITLE
1.16.2 discordsrv stop working 8.8.3

### ISSUE THREAD

**Issue by broken1arrow** [kangarko/ChatControl-Pro#2004]
https://bin.scarsz.me/9f6a114d-1e2a-4b4c-93ba-9f8df3bb3c3b#QHixt2xeihC80DP3oo06cikw9BD0DdKT

https://pastebin.com/UYq4JKyK

I think it was when i update too 8.8.3 DC stop sync MSG from minecraft chat to DC (it was in a 8.8.x update It work in 8.7).

**Comment by astirling** [comment]
I'm experiencing the same issue.  Chat messages are not being sent/received from Discord.  DiscordSRV is working as it still sends messages from in game to Discord for server start/stop, player advancements, etc.  My channel names/IDs have not changed in Discord or ChatControl.

**Comment by broken1arrow** [comment]
> 
> 
> I'm experiencing the same issue. Chat messages are not being sent/received between Discord and ChatControlPro. DiscordSRV is working as it still sends messages from in game to Discord for server start/stop, player advancements, etc. My channel names/IDs have not changed in Discord or ChatControl.

Yes, i have not change settings ether, it stop working when I update to newer version.

**Comment by kangarko** [comment]
Guys set Debug option in settings.yml to [discord] and report me your console messages when the message is sent but does not render.

**Comment by kangarko** [comment]
@astirling and @broken1arrow set Debug option in settings.yml to [discord] and report me your console messages when the message is sent but does not render.

**Comment by Sikatsu** [comment]
The new discordsrv update also makes ChatControl no longer enable on startup.

**Comment by broken1arrow** [comment]
> 
> 
> @astirling and @broken1arrow set Debug option in settings.yml to [discord] and report me your console messages when the message is sent but does not render.

It work now in your newest build 8.8.13

See when we get the discord boot update, so we can try newer build and see if I get same problem as Sikatsu

**Comment by kangarko** [comment]
Before I proceed, just to be clear, I have updated the docs: https://github.com/kangarko/ChatControl-Pro/wiki/Discord

- Also ensure to have Discord_Format option in there otherwise the message won't show.
- https://github.com/kangarko/ChatControl-Pro/wiki/Discord#important-before-continuing-set-debug-key-in-settingsyml-to-discord-and-restart-this-will-save-you-hours-and-print-out-what-is-happening-to-the-console-after-you-are-finished-with-your-setup-you-can-remove-the-value

Did you guys do it? What's the console output?

**Comment by kangarko** [comment]
> The new discordsrv update also makes ChatControl no longer enable on startup.

What's the error message and what release are you using?

For me it works just fine on 1.20.0.

![a](https://i.imgur.com/dm08IwI.png)

**Comment by Sikatsu** [comment]
Odd, re-tested it and now it booted. ^^

**Comment by astirling** [comment]
I was missing the 'Discord_Format' lines in my channels.yml.  Once I put those in chat was being received by Discord.  Thank you for pointing that out.  That wasn't a configuration option in previous versions (except for overall discord format in formatting.yml), at least that I was aware of.

**Comment by kangarko** [comment]
Yeah sorry for the confusion, making Discord_Format a requirement without docs was a let down, apology. In ChatControl 9 the Discord will work for channels by default if you connect the channel via settings.yml

Then you can use that connection anywhere else in the plugin. To explicitly turn it off from a channel there will be Discord option which you can set to "false". The Discord_Format will remain but if you don't set it, it will use the default one from settings.yml.

**Comment by astirling** [comment]
No worries.  I appreciate the plugin and the work that's put into it.  As an IT professional, I understand things falling in between the cracks. :)

**Comment by kangarko** [comment]
Thanks!

If you have exp in this field, I would love to hear your feedback:

https://www.spigotmc.org/resources/%E2%99%A3-chatcontrol-pro%E2%84%A2-format-and-filter-your-chat-1-7-10-1-16-3.10258/update?update=364984
